nodes/src/nodes/tool_git/IInstance.py

```
@@ -0,0 +1,432 @@
+        if op == 'tool.query':
+            for tool in _TOOLS:
+                param.tools.append(tool)
+            return param
+
+        if op == 'tool.invoke':
+            tool_name = getattr(param, 'tool_name', None)
+            args = getattr(param, 'input', {}) or {}
+            if isinstance(args, str):
+                try:
+                    args = json.loads(args)
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)